### PR TITLE
Plugin Manager Tests

### PR DESCRIPTION
**Pull request by ioedeveloper**

### CODE REVIEW

File: test-browser/tests/plugin.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    browser.waitForElementVisible('div[data-id="pluginManagerComponentPluginManager"]')
+    .click('button[data-id="pluginManagerSettingsButton"]')
+    .waitForElementVisible('form[data-id="pluginManagerSettingsPermissionForm"]')
+    .assert.containsText('form[data-id="pluginManagerSettingsPermissionForm"]', 'No Permission requested yet')
+    .modalFooterOKClick()
+    .click('div[data-id="verticalIconsFileExplorerIcons"]')
+    .switchFile('browser/3_Ballot.sol')
+    .click('div[plugin="ZoKrates"]')
+    .pause(5000)
+    .frame(0)
```
Comment (yann300) [inline test-browser/tests/plugin.js]: what is doing `frame` ?

Comment (ioedeveloper) [reply test-browser/tests/plugin.js]: `.frame(0)` allows switching into the first `<iframe>` indexed at 0 (plugin manager's iframe) and selecting DOM elements. I added this because i needed to select the compile button in ZoKrates which is inside an iframe.

File: test-browser/tests/pluginManager.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    .modalFooterOKClick()
+    .click('*[data-id="verticalIconsFileExplorerIcons"]')
+    .switchFile('browser/3_Ballot.sol')
+    .click('*[plugin="ZoKrates"]')
+    .pause(5000)
+    .frame(0)
+    .useXpath().click("//span[text()='Compile']")
+    .frameParent()
+    .useCss().waitForElementVisible('*[data-id="modalDialogContainer"]')
+    .assert.containsText('*[data-id="permissionHandlerMessage"]', 'ZOKRATES" WOULD LIKE TO ACCESS "FILE MANAGER" :')
```
Comment (yann300) [inline test-browser/tests/pluginManager.js]: we should also check is the api call is successful and returns the correct data, but that will be another PR...
